benchmark/script.py

```diff
@@ -244,6 +241,22 @@ def main(argv, environ, react=react):
     except UsageError as e:
         usage(options, e.args[0])
 
+    if options['log-file'] is not None:
+        try:
+            log_file = open(options['log-file'], 'a')
```

Comment:
I would actually specify 'w+', if the command was run twice in a row with the same log file name the user would expect the contents to be replaced, not appended to. At least that's my opinion. Change that and then merge if tests pass.